base-notebook/start.sh

```diff
@@ -39,12 +40,12 @@ run-hooks () {
 run-hooks /usr/local/bin/start-notebook.d
 
 # Handle special flags if we're root
-if [ $(id -u) == 0 ] ; then
+if [ "$(id -u)" == 0 ] ; then
```

Comment:
I think this will fail unless...

```suggestion
if [ "$(id -u)" == "0" ] ; then
```

Reply:
I created a simple script:

```
#!/bin/bash

if [ "$(id -u)" == 0 ] ; then
    echo "YES"
else
    echo "NO"
fi
```

It works both with or without quotes.

Reply:
Oh, okay